### PR TITLE
Update and improve metadata

### PR DESCRIPTION
**Pull request by razzeee**
Just some updates, as I was looking at https://blogs.gnome.org/tbernard/2021/09/07/ready-for-software-41/

I'll be looking at improving the screenshots too, but that might take some time.

### CODE REVIEW

File: tv.kodi.Kodi.metainfo.xml [hunk 1]
```diff
@@ -32,7 +32,42 @@
     </description>
     <launchable type="desktop-id">tv.kodi.Kodi.desktop</launchable>
     <screenshots>
-        <screenshot type="default">https://cdn.dday.it/system/uploads/news/main_image/22334/main_a1.jpg</screenshot>
+        <screenshot type="default">
+            <image>https://mirrors.kodi.tv/screenshots/kodi-recently-added.jpg</image>
+            <caption>The homescreen keeps your media organized</caption>
+        </screenshot>
+        <screenshot type="default">
```
Comment (AsciiWolf) [inline tv.kodi.Kodi.metainfo.xml:39]: Why do you have multiple default screenshots?

Comment (razzeee) [reply tv.kodi.Kodi.metainfo.xml:39]: According to the spec, this shouldn't be a problem, but your right, it's an oversight on my part.

Comment (razzeee) [reply tv.kodi.Kodi.metainfo.xml:39]: Removed this from upstream. Will change this MR to draft, till that's merged.
https://github.com/xbmc/xbmc/pull/20142

**Comment by razzeee** [comment]
I'm unsure how to proceed. Upstream merged the metainfo and will provide it with the next versions.

Maybe we'll just need to get rid of our hard coded one, but I can't test that, without a new version being released.

**Comment by wsnipex** [comment]
to use the upstream metainfo, you'll have to adjust APP_PACKAGE in version.txt

**Comment by basilgello** [comment]
Isn't there any pre-builtd hook or command similar to https://github.com/flathub/tv.kodi.Kodi/blob/master/tv.kodi.Kodi.yml#L83 ?
You can easily do `sed -i 's/org.xbmc.kodi/tv.kodi.Kodi/' version.txt` there.

Another way is to make another upstream PR allowing you to set `-DAPP_PACKAGE=tv.kodi.Kodi` in configure step and let that setting override the one from `version.txt` if needed.

**Comment by razzeee** [comment]
build failure is probably due to missing upstream changes, I'm currently trying to build with 

```diff
diff --git a/tv.kodi.Kodi.yml b/tv.kodi.Kodi.yml
index 3d156f6..89879b0 100644
--- a/tv.kodi.Kodi.yml
+++ b/tv.kodi.Kodi.yml
@@ -111,9 +111,8 @@ modules:
       - -DLIBDVDREAD_URL=build/download/libdvdread.tar.gz
       - -DLIBDVDNAV_URL=build/download/libdvdnav.tar.gz
     sources:
-      - type: archive
-        url: https://github.com/xbmc/xbmc/archive/19.2-Matrix.tar.gz
-        sha256: e8f5e50767ddd1b8f0399016dcf5ba762315c16c0568cb06e659c493376f99d5
+      - type: git
+        url: https://github.com/xbmc/xbmc.git
       - type: patch
         path: kodi.sh.in.patch
       - type: file
```

**Comment by razzeee** [comment]
Seems like this was already added in master

**Pull request closed without merging**